doc/write_your_own/connections/intro.rst

```diff
@@ -1,18 +1,27 @@
 Cube Browser Connections
 ========================
 
-This is a description of how you can pass information between widgets.
+Composing Connections
+---------------------
 
-Here are some examples of how you would connect them:
+For widgets which possess hard coded options, the only elements you need are the list of input options and the widget itself.
+Widgets that depend on the response of previous selections need:
+    - An initial list of input options
+    - The widget itself
+    - A function defining what change occurs if a selection occurs in an influencing widget
+    - A statement to observe the influencing widget and execute the function upon event of a change.
+
+While you are adding these elements to your notebook, you will also benefit from moving all your widgets into a single cell so that they can communicate with each other without having to re-run.
```

Comment:
This is in a different tense to other elements of the docs. You could also do with specifying what you're planning to re-run at the end of the sentence! How about:

"When adding these elements... a single cell. Doing so means they can... without having to re-run all the cells that contain widgets."

Note: it's not clear what 'these' in the above refers to. Are they connecting elements? Are they widgets that depend on the response of previous selections?